src/main/java/io/jenkins/pluginhealth/scoring/service/PluginService.java

```diff
@@ -56,4 +56,27 @@ public Plugin saveOrUpdate(Plugin plugin) {
     public Stream<Plugin> streamAll() {
         return pluginRepository.findAll().stream();
     }
+
+    @Transactional
+    public int getPluginsCount() {
+        return pluginRepository.findAll().size();
+    }
+
+    @Transactional
+    public int getProbeRawData(String probeID) {
+        switch (probeID) {
+            case "scm":
+            case "last-commit-date":
+            case "dependabot":
+            case "jep-229":
+            case "jenkinsfile":
+                return pluginRepository.getProbeRawData(probeID, "SUCCESS");
+            case "up-for-adoption":
+            case "security":
+            case "deprecation":
+                return pluginRepository.getProbeRawData(probeID, "FAILURE");
+            default:
+                return 0;
```

Comment:
This means that when we will be adding new probes in the future, we will always have to add it here. I don't think this is practical.

```suggestion
        switch (probeID) {
            case "up-for-adoption":
            case "security":
            case "deprecation":
                return pluginRepository.getProbeRawData(probeID, "FAILURE");
            default:
                return pluginRepository.getProbeRawData(probeID, "SUCCESS");
```

Reply:
Done, thanks